backend/src/services/UserService.js

```diff
@@ -0,0 +1,47 @@
+import UserRepository from '../repositories/UserRepository.js';
```

Comment:
Do we need this service anymore? It's same as UserRepository.

Reply:
Yes bhaia you are correct, I have refactored the code.